README.md

```diff
@@ -98,14 +98,31 @@ class MyFunction implements RequestHandler<String, String> {
     }
 ```
 
-### Support Java 11 and Above 
+### Support Java 11 and Above
 
 Add the environment variable `JAVA_TOOL_OPTIONS` to your Lambda functions and set it to
 `-Djdk.attach.allowAttachSelf=true` in addition to the manual code mentioned above.
 
 ### Supported Instrumentation Libraries
 
-- Aws SDK V1 
+- Aws SDK V1
 - Aws SDK V2
 - Apache HTTP Client
-- Apache Kafka
\ No newline at end of file
+- Apache Kafka
+
+### Secret scrubbing
+
+The tracer will automatically scrub values for keys matching the following regex patterns:
```

Comment:
Scrub keys from what? the word `payload` is after a long list of values, I think it should be in the first sentence

Reply:
Got point. See the fix below: